bindings/java/tools/build.py

```diff
@@ -55,22 +55,35 @@ def get_cargo_artifact_name(classifier: str) -> str:
     parser.add_argument('--target', type=str, default='')
     parser.add_argument('--profile', type=str, default='dev')
     parser.add_argument('--features', type=str, default='default')
+    parser.add_argument('--enable-zigbuild', type=str, default='false')
```

Comment:
I'm afraid that it becomes too complex. IIRC ASF INFRA is going to support arm runner and this can be an alternative to evaluate, instead of introduce complexity for cross compiling while the solution is not quite clean.

cc @PragmaTwice do you have the link where we can request for a arm runner now?

Reply:
> I'm afraid that it becomes too complex. IIRC ASF INFRA is going to support arm runner and this can be an alternative to evaluate, instead of introduce complexity for cross compiling while the solution is not quite clean.

It's not only about `aarch64` support but also glibc 2.26 support. Having a native aarch64 runner can't address the glibc issues.